charcoal/Snakefile

```diff
@@ -287,22 +287,23 @@ rule contigs_sig:
     """
 
 # run a search, query.x.database.
-rule search_all:
+rule prefetch_all:
     input:
         query = stage1_dir + '/{filename}.sig',
         databases = config['gather_db']
     output:
         csv = stage1_dir + '/{filename}.matches.csv',
-        matches = stage1_dir + '/{filename}.matches.sig',
+        matches = stage1_dir + '/{filename}.matches.zip',
         txt = stage1_dir + '/{filename}.matches.txt'
     params:
         moltype = "--{}".format(config['moltype'].lower()),
-        gather_scaled = config['gather_scaled']
+        gather_scaled = config['gather_scaled'],
+        threshold_bp = config['gather_scaled']*3
     conda: 'conf/env-sourmash.yml'
     shell: """
-        sourmash search {input.query} {input.databases} -o {output.csv} \
-            --containment {params.moltype} --scaled {params.gather_scaled} \
-            --save-matches {output.matches} --threshold=0.001 >& {output.txt}
+        sourmash prefetch --no-linear {input.query} {input.databases} -o {output.csv} \
```

Comment:
just as a note, `--no-linear` is the default here, so you don't need to specify it.